redshift_sqlalchemy/dialect.py

```diff
@@ -484,7 +487,8 @@ def get_unique_constraints(self, connection, table_name,
         """
         Return information about unique constraints in `table_name`.
 
-        See :meth:`~sqlalchemy.engine.interfaces.Dialect.get_unique_constraints`.
+        See
+        :meth:`~sqlalchemy.engine.interfaces.Dialect.get_unique_constraints`.
```

Comment:
If we build docs using sphinx, we can add sqlalchemy to the intersphinx mappings, which enables links like these. I created the basic structure for this in [a new sphinx branch of this repo](https://github.com/graingert/redshift_sqlalchemy/tree/sphinx). I was able to build those docs and test that these links work.